blacklist/blacklist/Common/Utils/AppConstants.swift

```diff
@@ -15,4 +15,8 @@ struct Constants {
         static let databaseName = "database.realm"
         static let rootDirectory = "opelty"
     }
+
+    struct Animations {
```

Comment:
We can set this Struct in another File, so we can handle all the animations there, I can do this work in other branch if you want.

Reply:
Like in a separate file? It sounds good to me.